scripts/pkginfo_to_json.g

```diff
@@ -3,14 +3,15 @@ LoadPackage("json");
 InstallMethod(RecNames, [IsRecord and IsInternalRep], x -> AsSSortedList(REC_NAMES(x)));
 
 InstallMethod(_GapToJsonStreamInternal, [IsOutputStream, IsObject],
-function(o, x)
+  function(o, x)
     PrintTo(o, "null");
-end);
+  end
+);
 
 Read("PackageInfo.g");
 if not IsBound(GAPInfo.PackageInfoCurrent) then
-  Print("Reading PackageInfo.g failed\n");
-  FORCE_QUIT_GAP(2);
+  Exec("echo \"::error::Reading PackageInfo.g failed\"");
```

Comment:
This file is a GAP "script", I mean, we can easily treat it as script with GAP as execution environment, but in general I'm not a big fan of including _Actions_ logic into GAP.

Writing an abstraction layer could be helpful.

@fingolfin What do you think?

Reply:
I guess we could have a file `gh_actions.g` which contains code that re-defines `Error` to do something else (e.g. call `Exec` like here), and then change e.g.
```
gap -A -q $GITHUB_ACTION_PATH/scripts/pkginfo_to_json.g
```
to
```
gap -A -q $GITHUB_ACTION_PATH/scripts/gh_actions.g $GITHUB_ACTION_PATH/scripts/pkginfo_to_json.g
```

But I am not sure it's worth the trouble? After all, you *can* run `pkginfo_to_json.g` locally even with that `Exec` for errors in it.